.github/workflows/workflow-roadmap.yml

```diff
@@ -1,4 +1,4 @@
-name: Update roadmap issue #728
+name: Update roadmap issues
```

Comment:
My only concern here is that we might hit GitHub API limits, particularly when we're triaging and there's a flurry of activity.

If we start hitting issues I think we have some options, some basic optimisations are probably possible, and potentially combining all updates into a single script, merging some calls.

Reply:
i'll combine these into a single workflow for now, since the triggers are the same